overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -414,9 +414,9 @@ func (s *servicectlSuite) TestQueuedCommandsSingleLane(c *C) {
 	defer s.st.Unlock()
 
 	laneTasks := chg.LaneTasks(0)
-	c.Assert(laneTasks, HasLen, 15)
-	c.Check(laneTasks[11].Summary(), Matches, `Run configure hook of .* snap if present`)
-	c.Check(laneTasks[12].Summary(), Equals, "stop of [test-snap.test-service]")
-	c.Check(laneTasks[13].Summary(), Equals, "start of [test-snap.test-service]")
-	c.Check(laneTasks[14].Summary(), Equals, "restart of [test-snap.test-service]")
+	c.Assert(laneTasks, HasLen, 16)
+	c.Check(laneTasks[12].Summary(), Matches, `Run configure hook of .* snap if present`)
+	c.Check(laneTasks[13].Summary(), Equals, "stop of [test-snap.test-service]")
+	c.Check(laneTasks[14].Summary(), Equals, "start of [test-snap.test-service]")
+	c.Check(laneTasks[15].Summary(), Equals, "restart of [test-snap.test-service]")
```

Comment:
Not for this PR, but the number shifting here gives me a feeling of irrelevance. We should eventually do something that doesn't involve so much wasteful manual fixing.